results differ between classic and mctc

I was seeing if I could drive mctc-gcp with slight modifications of the gcp QCEngine harness. atom is fine, but molecules differ in Egcp by about an order of magnitude with the same input. It's not a simple bohr/ang issue. I don't particularly need this solved -- just FYI and stashing notes.


```
command=['mctc-gcp', 'gcp_geometry.xyz', '-level', 'HF/SV', '-grad']
infiles={'gcp_geometry.xyz': '10\n0 1 C4H6\nC                     0.000000000000    -0.667578000000    -2.124659000000\nC                     0.000000000000     0.667578000000    -2.124659000000\nH                     0.923621000000    -1.232253000000    -2.126185000000\nH                    -0.923621000000    -1.232253000000    -2.126185000000\nH                    -0.923621000000     1.232253000000    -2.126185000000\nH                     0.923621000000     1.232253000000    -2.126185000000\nC                     0.000000000000     0.000000000000     2.900503000000\nC                     0.000000000000     0.000000000000     1.693240000000\nH                     0.000000000000     0.000000000000     0.627352000000\nH                     0.000000000000     0.000000000000     3.963929000000\n'}
```


"classic" on left, mctc on right
```
'keywords': {},																	 'keywords': {},
 'model': {'basis': None, 'method': 'hf/sv'},														 'model': {'basis': None, 'method': 'hf/sv'},
 'molecule': {'atom_labels': array(['', '', '', '', '', '', '', '', '', ''], dtype='<U1'),								 'molecule': {'atom_labels': array(['', '', '', '', '', '', '', '', '', ''], dtype='<U1'),
              'atomic_numbers': array([6, 6, 1, 1, 1, 1, 6, 6, 1, 1], dtype=int16),									              'atomic_numbers': array([6, 6, 1, 1, 1, 1, 6, 6, 1, 1], dtype=int16),
              'fix_com': False,																              'fix_com': False,
              'fix_orientation': False,															              'fix_orientation': False,
              'fragment_charges': [0.0, 0.0],														              'fragment_charges': [0.0, 0.0],
              'fragment_multiplicities': [1, 1],													              'fragment_multiplicities': [1, 1],
              'fragments': [array([0, 1, 2, 3, 4, 5], dtype=int32),											              'fragments': [array([0, 1, 2, 3, 4, 5], dtype=int32),
                            array([6, 7, 8, 9], dtype=int32)],												                            array([6, 7, 8, 9], dtype=int32)],
              'geometry': array([[ 0.        , -1.26153959, -4.01502362],										              'geometry': array([[ 0.        , -1.26153959, -4.01502362],
       [ 0.        ,  1.26153959, -4.01502362],														       [ 0.        ,  1.26153959, -4.01502362],
       [ 1.74539073, -2.32862069, -4.01790734],														       [ 1.74539073, -2.32862069, -4.01790734],
       [-1.74539073, -2.32862069, -4.01790734],														       [-1.74539073, -2.32862069, -4.01790734],
       [-1.74539073,  2.32862069, -4.01790734],														       [-1.74539073,  2.32862069, -4.01790734],
       [ 1.74539073,  2.32862069, -4.01790734],														       [ 1.74539073,  2.32862069, -4.01790734],
       [ 0.        ,  0.        ,  5.4811563 ],														       [ 0.        ,  0.        ,  5.4811563 ],
       [ 0.        ,  0.        ,  3.19975986],														       [ 0.        ,  0.        ,  3.19975986],
       [ 0.        ,  0.        ,  1.18552346],														       [ 0.        ,  0.        ,  1.18552346],
       [ 0.        ,  0.        ,  7.49074019]]),													       [ 0.        ,  0.        ,  7.49074019]]),
              'mass_numbers': array([12, 12,  1,  1,  1,  1, 12, 12,  1,  1], dtype=int16),								              'mass_numbers': array([12, 12,  1,  1,  1,  1, 12, 12,  1,  1], dtype=int16),
              'masses': array([12.        , 12.        ,  1.00782503,  1.00782503,  1.00782503,								              'masses': array([12.        , 12.        ,  1.00782503,  1.00782503,  1.00782503,
        1.00782503, 12.        , 12.        ,  1.00782503,  1.00782503]),										        1.00782503, 12.        , 12.        ,  1.00782503,  1.00782503]),
              'molecular_charge': 0.0,															              'molecular_charge': 0.0,
              'molecular_multiplicity': 1,														              'molecular_multiplicity': 1,
              'name': 'C4H6',																              'name': 'C4H6',
              'provenance': {'creator': 'QCElemental',													              'provenance': {'creator': 'QCElemental',
                             'routine': 'qcelemental.molparse.from_string',										                             'routine': 'qcelemental.molparse.from_string',
                             'version': 'v0.18.0'},													                             'version': 'v0.18.0'},
              'real': array([ True,  True,  True,  True,  True,  True,  True,  True,  True,								              'real': array([ True,  True,  True,  True,  True,  True,  True,  True,  True,
        True]),																		        True]),
              'schema_name': 'qcschema_molecule',													              'schema_name': 'qcschema_molecule',
              'schema_version': 2,															              'schema_version': 2,
              'symbols': array(['C', 'C', 'H', 'H', 'H', 'H', 'C', 'C', 'H', 'H'], dtype='<U1'),							              'symbols': array(['C', 'C', 'H', 'H', 'H', 'H', 'C', 'C', 'H', 'H'], dtype='<U1'),
              'validated': True},															              'validated': True},
 'properties': {},																	 'properties': {},
 'protocols': {},																	 'protocols': {},
 'provenance': {'cpu': 'Intel(R) Xeon(R) CPU E5-2630 v4 @ 2.20GHz',											 'provenance': {'cpu': 'Intel(R) Xeon(R) CPU E5-2630 v4 @ 2.20GHz',
                'creator': 'GCP',															                'creator': 'GCP',
                'hostname': 'psinet',															                'hostname': 'psinet',
                'qcengine_version': 'v0.18.0+12.g30ed0d2.dirty',											                'qcengine_version': 'v0.18.0+12.g30ed0d2.dirty',
                'routine': 'qcengine.programs.gcp.parse_output',											                'routine': 'qcengine.programs.gcp.parse_output',
                'username': 'psilocaluser',														                'username': 'psilocaluser',
                'version': '2.2',														     |	                'version': '2.1.2',
                'wall_time': 1.2454249858856201},												     |	                'wall_time': 1.2334864139556885},
 'return_result': array([[-6.11830758e-20, -6.91724066e-05,  1.09826314e-04],									     |	 'return_result': array([[ 2.33645568e-21,  1.88406240e-04,  9.40795164e-05],
       [ 7.12044626e-20,  6.91724066e-05,  1.09826314e-04],											     |	       [-4.67291136e-21, -1.88406240e-04,  9.40795164e-05],
       [-1.52546975e-03,  1.10309566e-03,  2.80148912e-05],											     |	       [ 2.63848984e-05,  6.77932735e-05,  2.46892935e-05],
       [ 1.52546975e-03,  1.10309566e-03,  2.80148912e-05],											     |	       [-2.63848984e-05,  6.77932735e-05,  2.46892935e-05],
       [ 1.52546975e-03, -1.10309566e-03,  2.80148912e-05],											     |	       [-2.63848984e-05, -6.77932735e-05,  2.46892935e-05],
       [-1.52546975e-03, -1.10309566e-03,  2.80148912e-05],											     |	       [ 2.63848984e-05, -6.77932735e-05,  2.46892935e-05],
       [-2.77691314e-24, -5.08629086e-24,  1.82372296e-04],											     |	       [ 1.14084750e-23,  6.84508500e-23, -1.57281683e-04],
       [-6.06354197e-23, -1.02697054e-21, -2.59507279e-04],											     |	       [ 3.65071200e-23,  1.82535600e-22,  7.77370261e-05],
       [-4.59899866e-22,  3.60801817e-21,  1.47563104e-03],											     |	       [ 0.00000000e+00,  0.00000000e+00, -2.02905480e-04],
       [ 3.74257354e-26, -2.25113303e-25, -1.73020825e-03]]),											     |	       [ 0.00000000e+00, -1.14084750e-24, -4.46606984e-06]]),
 'schema_name': 'qcschema_output',															 'schema_name': 'qcschema_output',
 'schema_version': 1,																	 'schema_version': 1,
 'stderr': None,																	 'stderr': None,
 'stdout': '  ___________________________________________ \n'											     |	 'stdout': ' -------------------------------------------\n'
           '                                              \n'											     <
           ' |              **  g C P  **                |\n'												           '|              **  g C P  **                |\n'
           ' |  a geometrical counterpoise correction    |\n'												           '|  a geometrical counterpoise correction    |\n'
           ' |     H.Kruse J.G.Brandenburg S.Grimme      |\n'												           '|     H.Kruse J.G.Brandenburg S.Grimme      |\n'
           ' |          Version 2.02 Nov 2016            |\n'											     |	           ' -------------------------------------------\n'
           ' |                                           |\n'											     |	           'mctc-gcp version 2.1.2\n'
           '  ___________________________________________ \n'											     <
           ' \n'																	           '\n'
           ' Please cite work done with this code as:\n'												           'Please cite work done with this code as:\n'
           ' H. Kruse, S. Grimme J. Chem. Phys. 136, 154101 (2012)\n'											           'H. Kruse, S. Grimme J. Chem. Phys. 136, 154101 (2012)\n'
           ' DOI: 10.1063/1.3700154 \n'															           'DOI: 10.1063/1.3700154\n'
           ' For the periodic version, please also cite:\n'												           'For the periodic version, please also cite:\n'
           ' J. G. Brandenburg, M. Alessio, B. Civalleri, M. F. Peintinger\n'										           'J. G. Brandenburg, M. Alessio, B. Civalleri, M. F. Peintinger\n'
           ' T. Bredow, S.Grimme J. Phys. Chem. A 117, 9282-9292 (2013).\n'										           'T. Bredow, S.Grimme J. Phys. Chem. A 117, 9282-9292 (2013).\n'
           ' DOI: 10.1021/jp406658y \n'															           'DOI: 10.1021/jp406658y\n'
           ' \n'																	           '\n'
           ' \n'																	           ' \n'
           ' reading...      XYZ file [angst]: gcp_geometry.xyz\n'										     <
           ' \n'																     <
           '  level           hf/sv  basis: SV (Ahlrichs)\n'												           '  level           hf/sv  basis: SV (Ahlrichs)\n'
           '  Nbf                48\n'															           '  Nbf                48\n'
           '  Atoms              10\n'															           '  Atoms              10\n'
           ' \n'																	           ' \n'
           '  Parameters: \n'																           '  Parameters: \n'
           '  sigma         0.1724\n'															           '  sigma         0.1724\n'
           '  eta           1.2804\n'															           '  eta           1.2804\n'
           '  alpha         0.8568\n'															           '  alpha         0.8568\n'
           '  beta          1.2342\n'															           '  beta          1.2342\n'
																		     >	           '  dmp_scal      4.0000\n'
																		     >	           '  dmp_exp       6.0000\n'
           ' \n'																	           ' \n'
           '  \n'																	           '  \n'
           ' element parameters hf/sv\n'														           ' element parameters hf/sv\n'
           '  elem   emiss   nbas    elem   emiss   nbas    elem   emiss   '										           '  elem   emiss   nbas    elem   emiss   nbas    elem   emiss   '
           'nbas\n'																	           'nbas\n'
           '    h    0.00904    2      he   0.00884    2      li   0.20419    '										           '    h    0.00904    2      he   0.00884    2      li   0.20419    '
           '3\n'																	           '3\n'
           '    be   0.10775    3      b    0.04953    9      c    0.05548    '										           '    be   0.10775    3      b    0.04953    9      c    0.05548    '
           '9\n'																	           '9\n'
           '    n    0.07282    9      o    0.10085    9      f    0.13403    '										           '    n    0.07282    9      o    0.10085    9      f    0.13403    '
           '9\n'																	           '9\n'
           '    ne   0.17422    9      na   0.31562    7      mg   0.26112    '										           '    ne   0.17422    9      na   0.31562    7      mg   0.26112    '
           '7\n'																	           '7\n'
           '    al   0.16857   13      si   0.15229   13      p    0.14691   '										           '    al   0.16857   13      si   0.15229   13      p    0.14691   '
           '13\n'																	           '13\n'
           '    s    0.16825   13      cl   0.18788   13      ar   0.21116   '										           '    s    0.16825   13      cl   0.18788   13      ar   0.21116   '
           '13\n'																	           '13\n'
           '    k    0.37425   11      ca   0.46097   11      sc   0.44489   '										           '    k    0.37425   11      ca   0.46097   11      sc   0.44489   '
           '21\n'																	           '21\n'
           '    ti   0.40499   21      v    0.37841   21      cr   0.37344   '										           '    ti   0.40499   21      v    0.37841   21      cr   0.37344   '
           '21\n'																	           '21\n'
           '    mn   0.36125   21      fe   0.36001   21      co   0.36293   '										           '    mn   0.36125   21      fe   0.36001   21      co   0.36293   '
           '21\n'																	           '21\n'
           '    ni   0.24380   21      cu   0.40530   21      zn   0.39651   '										           '    ni   0.24380   21      cu   0.40530   21      zn   0.39651   '
           '21\n'																	           '21\n'
           '    ga   0.36267   27      ge   0.36046   27      as   0.36336   '										           '    ga   0.36267   27      ge   0.36046   27      as   0.36336   '
           '27\n'																	           '27\n'
           '    se   0.38417   27      br   0.39970   27      kr   0.41731   '										           '    se   0.38417   27      br   0.39970   27      kr   0.41731   '
           '27\n'																	           '27\n'
           '  \n'																	           '  \n'
           '   \n'																	           '   \n'
           ' cutoffs: \n'																           ' cutoffs: \n'
           '   distance [bohr] 60.0\n'															           '   distance [bohr] 60.0\n'
           '   noise    [a.u.] 2.2E-16\n'														           '   noise    [a.u.] 2.2E-16\n'
           '   SR-damping      F\n'														     |	           '   SR-damping      T\n'
           '   \n'																	           '   \n'
           '      #     ON  sites    Nvirt           Emiss    BSSE [kcal/mol]\n'									           '      #     ON  sites    Nvirt           Emiss    BSSE [kcal/mol]\n'
           '      1      6      9      6.0          0.0555          2.8760\n'									     |	           '      1      6      9      6.0          0.0555          0.3915\n'
           '      2      6      9      6.0          0.0555          2.8760\n'									     |	           '      2      6      9      6.0          0.0555          0.3915\n'
           '      3      1      9      1.5          0.0090          0.1846\n'									     |	           '      3      1      9      1.5          0.0090          0.0729\n'
           '      4      1      9      1.5          0.0090          0.1846\n'									     |	           '      4      1      9      1.5          0.0090          0.0729\n'
           '      5      1      9      1.5          0.0090          0.1846\n'									     |	           '      5      1      9      1.5          0.0090          0.0729\n'
           '      6      1      9      1.5          0.0090          0.1846\n'									     |	           '      6      1      9      1.5          0.0090          0.0729\n'
           '      7      6      9      6.0          0.0555          1.6288\n'									     |	           '      7      6      9      6.0          0.0555          0.1542\n'
           '      8      6      9      6.0          0.0555          1.6586\n'									     |	           '      8      6      9      6.0          0.0555          0.1870\n'
           '      9      1      9      1.5          0.0090          0.1346\n'									     |	           '      9      1      9      1.5          0.0090          0.0455\n'
           '     10      1      9      1.5          0.0090          0.1033\n'									     |	           '     10      1      9      1.5          0.0090          0.0154\n'
           'cpu-time for gradient    0.0 s\n'														           'cpu-time for gradient    0.0 s\n'
           '                 \n'															           '\n'
           ' ** gCP correction ** \n'															           '** gCP correction ** \n'
           '  Egcp:        0.0159608596 / (a.u.) ||      10.0156 / (kcal/mol)\n'								     |	           '  Egcp:        0.0023531419 / (a.u.) ||       1.4766 / (kcal/mol)\n'
           '                 \n'															           '\n'
           ' |G|=  1.463203748874275E-002\n'													     |	           ' |G|=   1.4828316340119714E-003\n'
           ' no gradient file found to add G(gcp)!\n'													           ' no gradient file found to add G(gcp)!\n'
           ' hence written to file gcp_gradient\n'												     |	           ' hence written to file gcp_gradient\n',
           'cpu-time for gCP    0.0 s\n'													     <
           ' \n'																     <
           '     normal termination of gCP\n',													     <
 'success': True,																	 'success': True,
 'wavefunction': None}																	 'wavefunction': None}
{'CURRENT ENERGY': '0.0159608596', 'GCP CORRECTION ENERGY': '0.0159608596', 'HF/SV GCP CORRECTION ENERGY': '0.0159608596', 'CURRENT GRADIENT': [-6.1 |	{'CURRENT ENERGY': '0.0023531419', 'GCP CORRECTION ENERGY': '0.0023531419', 'HF/SV GCP CORRECTION ENERGY': '0.0023531419', 'CURRENT GRADIENT': [2.33
==================================================================================================================================================== |	----------------------------------------------------------------------------------------------------------------------------------------------------
FAILED qcengine/programs/tests/test_dftd3_mp2d.py::test_gcp[inp0-qcmol] - assert 0								     |	ERROR    root:testing.py:20     test_gcp.............................................FAILED
																		     >	ERROR    root:testing.py:21         test_gcp: computed value (0.002353142) does not match (0.015960860) to atol=1e-07 by difference (-0.013607718).
```

--- a/.github/workflows/build.yml
+++ b/.github/workflows/build.yml
@@ -37,7 +37,7 @@ jobs:
         --slave /usr/bin/gcov gcov /usr/bin/gcov-${GCC_V}
 
     - name: Install meson
-      run: pip3 install meson ninja
+      run: pip3 install meson==0.56.2 ninja
 
     - name: Configure build
       run: meson setup _build

--- a/meson.build
+++ b/meson.build
@@ -17,7 +17,7 @@
 project(
   'mctc-gcp',
   'fortran',
-  version: '2.1.2',
+  version: '2.3.0',
   license: 'GPL-3.0-or-later',
   meson_version: '>=0.53',
   default_options: [
@@ -56,6 +56,7 @@ gcp_dep = declare_dependency(
 
 # Add applications
 subdir('app')
+subdir('old')
 
 # Package the license files
 gcp_lic = files(

--- a/old/main.f90
+++ b/old/main.f90
@@ -99,7 +99,7 @@ write(*,*)'                                             '
 write(*,*)'|              **  g C P  **                |'
 write(*,*)'|  a geometrical counterpoise correction    |'
 write(*,*)'|     H.Kruse J.G.Brandenburg S.Grimme      |'
-write(*,*)'|          Version 2.02 Nov 2016            |'
+write(*,*)'|          Version 2.3.0 Feb 2021           |'
 write(*,*)'|                                           |'
 write(*,*)' ___________________________________________ '
 write(*,*)

--- a/old/meson.build
+++ b/old/meson.build
@@ -1,0 +1,22 @@
+# This file is part of mctc-gcp.
+# SPDX-Identifier: GPL-3.0-or-later
+#
+# mctc-gcp is free software: you can redistribute it and/or modify it under
+# the terms of the GNU General Public License as published by
+# the Free Software Foundation, either version 3 of the License, or
+# (at your option) any later version.
+#
+# mctc-gcp is distributed in the hope that it will be useful,
+# but WITHOUT ANY WARRANTY; without even the implied warranty of
+# MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE.  See the
+# GNU General Public License for more details.
+#
+# You should have received a copy of the GNU General Public License
+# along with mctc-gcp.  If not, see <https://www.gnu.org/licenses/>.
+
+gcp_exe = executable(
+  'gcp',
+  sources: files('main.f90'),
+  dependencies: gcp_dep,
+  install: false,
+)

--- a/src/gcp.f90
+++ b/src/gcp.f90
@@ -72,7 +72,7 @@ dmp_scal=4.0d0  ! 4
 dmp_exp=6.0d0   ! 6
 !default
 base=.false.
-damp=.true.
+damp=.false.
 
 !************************
 !* modify input string  *
@@ -109,6 +109,8 @@ if(echo) write(*,*) ' Perform special SRB correction for B97-3c '
 srb=.true.
 rscal=10.00d0
 qscal=0.08d0
+case('r2scan3c','def2mtzvpp','mtzvpp')
+damp=.true.
 case default
 end select
 

--- a/src/gcp_version.f90
+++ b/src/gcp_version.f90
@@ -24,10 +24,10 @@ module gcp_version
 
 
    !> String representation of the mctc-gcp version
-   character(len=*), parameter :: gcp_version_string = "2.1.2"
+   character(len=*), parameter :: gcp_version_string = "2.3.0"
 
    !> Numeric representation of the mctc-gcp version
-   integer, parameter :: gcp_version_compact(3) = [2, 1, 2]
+   integer, parameter :: gcp_version_compact(3) = [2, 3, 0]
 
 
 contains

--- a/test/test_gcp_3c_methods.f90
+++ b/test/test_gcp_3c_methods.f90
@@ -101,7 +101,7 @@ subroutine test_hf3c(error)
    type(structure_type) :: mol
 
    call get_structure(mol, "MB16-43", "01")
-   call test_generic(error, mol, "hf3c", -0.10185940933506053_wp)
+   call test_generic(error, mol, "hf3c", -3.0475695153528881E-2_wp)
 
 end subroutine test_hf3c
 
@@ -166,7 +166,7 @@ subroutine test_hf3c_pbc(error)
    type(structure_type) :: mol
 
    call get_structure(mol, "X23", "acetic")
-   call test_generic(error, mol, "hf3c", -0.39914501373759642_wp)
+   call test_generic(error, mol, "hf3c", -0.21073394289693193_wp)
 
 end subroutine test_hf3c_pbc
 

--- a/test/test_gcp_dft_methods.f90
+++ b/test/test_gcp_dft_methods.f90
@@ -101,7 +101,7 @@ subroutine test_dft_minis(error)
    type(structure_type) :: mol
 
    call get_structure(mol, "MB16-43", "13")
-   call test_generic(error, mol, "dft/minis", 2.8831779044393737E-2_wp)
+   call test_generic(error, mol, "dft/minis", 1.5921613343027519E-1_wp)
 
 end subroutine test_dft_minis
 
@@ -114,7 +114,7 @@ subroutine test_dft_minix(error)
    type(structure_type) :: mol
 
    call get_structure(mol, "MB16-43", "14")
-   call test_generic(error, mol, "dft/minix", 1.8191237759356119E-2_wp)
+   call test_generic(error, mol, "dft/minix", 1.0873902337093687E-1_wp)
 
 end subroutine test_dft_minix
 
@@ -127,7 +127,7 @@ subroutine test_dft_sv(error)
    type(structure_type) :: mol
 
    call get_structure(mol, "MB16-43", "15")
-   call test_generic(error, mol, "dft/sv", 8.9178105933847233E-3_wp)
+   call test_generic(error, mol, "dft/sv", 4.2061458290263622E-002_wp)
 
 end subroutine test_dft_sv
 
@@ -140,7 +140,7 @@ subroutine test_dft_def2sv_p(error)
    type(structure_type) :: mol
 
    call get_structure(mol, "MB16-43", "16")
-   call test_generic(error, mol, "dft/def2-sv(p)", 9.0695660381885229E-3_wp)
+   call test_generic(error, mol, "dft/def2-sv(p)", 3.6732928803962006E-2_wp)
 
 end subroutine test_dft_def2sv_p
 
@@ -153,7 +153,7 @@ subroutine test_dft_def2svp(error)
    type(structure_type) :: mol
 
    call get_structure(mol, "MB16-43", "17")
-   call test_generic(error, mol, "dft/def2-svp", 2.4834643199826180E-2_wp)
+   call test_generic(error, mol, "dft/def2-svp", 6.8082018400795294E-2_wp)
 
 end subroutine test_dft_def2svp
 
@@ -166,7 +166,7 @@ subroutine test_dft_dzp(error)
    type(structure_type) :: mol
 
    call get_structure(mol, "MB16-43", "18")
-   call test_generic(error, mol, "dft/dzp", 2.2195075119243783E-2_wp)
+   call test_generic(error, mol, "dft/dzp", 6.0859678618522664E-2_wp)
 
 end subroutine test_dft_dzp
 
@@ -179,7 +179,7 @@ subroutine test_dft_deftzvp(error)
    type(structure_type) :: mol
 
    call get_structure(mol, "MB16-43", "19")
-   call test_generic(error, mol, "dft/def-tzvp", 3.6258064909241677E-3_wp)
+   call test_generic(error, mol, "dft/def-tzvp", 1.0946847699932089E-2_wp)
 
 end subroutine test_dft_deftzvp
 
@@ -192,7 +192,7 @@ subroutine test_dft_def2tzvp(error)
    type(structure_type) :: mol
 
    call get_structure(mol, "MB16-43", "20")
-   call test_generic(error, mol, "dft/def2-tzvp", 3.9044561047188592E-3_wp)
+   call test_generic(error, mol, "dft/def2-tzvp", 9.8496583133859551E-3_wp)
 
 end subroutine test_dft_def2tzvp
 
@@ -205,7 +205,7 @@ subroutine test_dft_ccpvdz(error)
    type(structure_type) :: mol
 
    call get_structure(mol, "MB16-43", "21")
-   call test_generic(error, mol, "dft/cc-pvdz", 9.9649294926933245E-3_wp)
+   call test_generic(error, mol, "dft/cc-pvdz", 3.3952526484868496E-2_wp)
 
 end subroutine test_dft_ccpvdz
 
@@ -218,7 +218,7 @@ subroutine test_dft_augccpvdz(error)
    type(structure_type) :: mol
 
    call get_structure(mol, "MB16-43", "22")
-   call test_generic(error, mol, "dft/aug-cc-pvdz", 2.9849348931724447E-3_wp)
+   call test_generic(error, mol, "dft/aug-cc-pvdz", 5.9838861620780713E-3_wp)
 
 end subroutine test_dft_augccpvdz
 

--- a/test/test_gcp_hf_methods.f90
+++ b/test/test_gcp_hf_methods.f90
@@ -101,7 +101,7 @@ subroutine test_hf_minis(error)
    type(structure_type) :: mol
 
    call get_structure(mol, "MB16-43", "04")
-   call test_generic(error, mol, "hf/minis", 1.5336656998809515E-2_wp)
+   call test_generic(error, mol, "hf/minis", 7.6480869740597812E-2_wp)
 
 end subroutine test_hf_minis
 
@@ -114,7 +114,7 @@ subroutine test_hf_minix(error)
    type(structure_type) :: mol
 
    call get_structure(mol, "MB16-43", "05")
-   call test_generic(error, mol, "hf/minix", 1.8559832674618637E-2_wp)
+   call test_generic(error, mol, "hf/minix", 1.0104364899718374E-1_wp)
 
 end subroutine test_hf_minix
 
@@ -127,7 +127,7 @@ subroutine test_hf_sv(error)
    type(structure_type) :: mol
 
    call get_structure(mol, "MB16-43", "06")
-   call test_generic(error, mol, "hf/sv", 6.4701104644333107E-3_wp)
+   call test_generic(error, mol, "hf/sv", 2.6617114832609839E-2_wp)
 
 end subroutine test_hf_sv
 
@@ -140,7 +140,7 @@ subroutine test_hf_def2sv_p(error)
    type(structure_type) :: mol
 
    call get_structure(mol, "MB16-43", "07")
-   call test_generic(error, mol, "hf/def2-sv(p)", 1.5009548340274959E-2_wp)
+   call test_generic(error, mol, "hf/def2-sv(p)", 3.8517711151713638E-2_wp)
 
 end subroutine test_hf_def2sv_p
 
@@ -153,7 +153,7 @@ subroutine test_hf_def2svp(error)
    type(structure_type) :: mol
 
    call get_structure(mol, "MB16-43", "08")
-   call test_generic(error, mol, "hf/def2-svp", 1.8249078757662872E-2_wp)
+   call test_generic(error, mol, "hf/def2-svp", 3.7848835788317346E-2_wp)
 
 end subroutine test_hf_def2svp
 
@@ -166,7 +166,7 @@ subroutine test_hf_dzp(error)
    type(structure_type) :: mol
 
    call get_structure(mol, "MB16-43", "08")
-   call test_generic(error, mol, "hf/dzp", 2.4002048454141937E-2_wp)
+   call test_generic(error, mol, "hf/dzp", 4.3822089521084677E-2_wp)
 
 end subroutine test_hf_dzp
 
@@ -179,7 +179,7 @@ subroutine test_hf_deftzvp(error)
    type(structure_type) :: mol
 
    call get_structure(mol, "MB16-43", "09")
-   call test_generic(error, mol, "hf/def-tzvp", 4.9799363418100567E-3_wp)
+   call test_generic(error, mol, "hf/def-tzvp", 1.4409205281517200E-2_wp)
 
 end subroutine test_hf_deftzvp
 
@@ -192,7 +192,7 @@ subroutine test_hf_def2tzvp(error)
    type(structure_type) :: mol
 
    call get_structure(mol, "MB16-43", "10")
-   call test_generic(error, mol, "hf/def2-tzvp", 2.1614549442281130E-3_wp)
+   call test_generic(error, mol, "hf/def2-tzvp", 5.7102357181177616E-3_wp)
 
 end subroutine test_hf_def2tzvp
 
@@ -205,7 +205,7 @@ subroutine test_hf_ccpvdz(error)
    type(structure_type) :: mol
 
    call get_structure(mol, "MB16-43", "11")
-   call test_generic(error, mol, "hf/cc-pvdz", 9.5654476866715681E-3_wp)
+   call test_generic(error, mol, "hf/cc-pvdz", 2.8850619018747792E-2_wp)
 
 end subroutine test_hf_ccpvdz
 
@@ -218,7 +218,7 @@ subroutine test_hf_augccpvdz(error)
    type(structure_type) :: mol
 
    call get_structure(mol, "MB16-43", "12")
-   call test_generic(error, mol, "hf/aug-cc-pvdz", 2.6289439007561133E-3_wp)
+   call test_generic(error, mol, "hf/aug-cc-pvdz", 4.8150616062504351E-3_wp)
 
 end subroutine test_hf_augccpvdz
 

--- a/test/test_gcp_pbc.f90
+++ b/test/test_gcp_pbc.f90
@@ -98,7 +98,7 @@ subroutine test_anthracene(error)
    type(structure_type) :: mol
 
    call get_structure(mol, "X23", "anthracene")
-   call test_generic(error, mol, "hf3c", -0.92569438699860529_wp)
+   call test_generic(error, mol, "hf3c", -0.59161990455181912_wp)
 
 end subroutine test_anthracene
 
@@ -111,7 +111,7 @@ subroutine test_ethcar(error)
    type(structure_type) :: mol
 
    call get_structure(mol, "X23", "ethcar")
-   call test_generic(error, mol, "hf3c", -0.26393725722798922_wp)
+   call test_generic(error, mol, "hf3c", -0.11814220512782711_wp)
 
 end subroutine test_ethcar
 
@@ -124,7 +124,7 @@ subroutine test_formamide(error)
    type(structure_type) :: mol
 
    call get_structure(mol, "X23", "formamide")
-   call test_generic(error, mol, "hf3c", -0.29006658275561281_wp)
+   call test_generic(error, mol, "hf3c", -0.15500632261441180_wp)
 
 end subroutine test_formamide
 
@@ -137,7 +137,7 @@ subroutine test_hexdio(error)
    type(structure_type) :: mol
 
    call get_structure(mol, "X23", "hexdio")
-   call test_generic(error, mol, "hf3c", -0.18028851452622813_wp)
+   call test_generic(error, mol, "hf3c", -4.4642369521389225E-002_wp)
 
 end subroutine test_hexdio
 
@@ -150,7 +150,7 @@ subroutine test_naph(error)
    type(structure_type) :: mol
 
    call get_structure(mol, "X23", "naph")
-   call test_generic(error, mol, "hf3c", -1.3397571928969214_wp)
+   call test_generic(error, mol, "hf3c", -1.0399600000679481_wp)
 
 end subroutine test_naph
 
@@ -163,7 +163,7 @@ subroutine test_oxacb(error)
    type(structure_type) :: mol
 
    call get_structure(mol, "X23", "oxacb")
-   call test_generic(error, mol, "hf3c", -0.27269534244235344_wp)
+   call test_generic(error, mol, "hf3c", -0.15716399742412498_wp)
 
 end subroutine test_oxacb
 

